src/actions/index.js

```diff
@@ -11,4 +12,9 @@ const removeBook = (id) => ({
   id,
 });
 
-export { createBook, removeBook };
+const changeFilter = (id) => ({
+  type: CHANGE_FILTER,
+  id,
+});
```

Comment:
- The reducer would receive `action.id` not `action.book` as you used. Simply change the reducer to return `action.id`.